### PR TITLE
Docs/macos docker brew resources

### LINKED ISSUE #1162: Getting Started Tutorial Mac - Move X Code to Pre-requisite section.
Running through the tutorial with Mac, as is.. User experiences a long hang time with Yarn Install (20 mins +).. Reading the snippet that is there, its easy to decipher that we need to ensure Xcode is installed. If that is the case we sould not await the user to fail, then search through debugging notes, we should simply list Xcode as a requirement.  I Understand that most "developers" will have Xcode, however it may be helpful to think about "users" as well as "developers". 

Just thinking...

### PR DESCRIPTION
**Pull request by Jovonni**
addresses #1161

### CODE REVIEW

**Comment by dckc** [comment]
I don't do much development on a mac; deferring to @0xpatrickdev

File: yarn.lock [hunk 1]
```diff
@@ -5681,4 +5681,4 @@ yargs@^17.5.1:
 yocto-queue@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-1.0.0.tgz#7f816433fb2cbc511ec8bf7d263c3b58a1a3c251"
-  integrity sha512-9bnSc/HEW2uRy67wc+T8UwauLuPJVn28jb+GtJY16iiKWyvmYJRXVT4UamsAEGQfPohgr2q4Tq0sQbQlxTfi1g==
+  integrity sha512-9bnSc/HEW2uRy67wc+T8UwauLuPJVn28jb+GtJY16iiKWyvmYJRXVT4UamsAEGQfPohgr2q4Tq0sQbQlxTfi1g==
```
Comment (dckc) [inline yarn.lock:5684]: spurious change?

**Review by aj-agoric — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (Jovonni) [reply yarn.lock:5684]: very spurious 🫠

**Review by amessbee — APPROVED** [review]
@Jovonni LGTM - let us merge it!

**Review by 0xpatrickdev — APPROVED** [review]
This LGTM - we have similar changes in #1160 - but we can close that.

Before merging please these two changes present in 1160 but not here - 

- change the second `[installation for Ubuntu]` to `[installation for Mac]` (we have installation for ubuntu 2x)
- move the group instructions to the Ubuntu `<details>` - we only need to do this on ubuntu, not mac

**Comment by Jovonni** [comment]
moved instructions to ubuntu details:
https://github.com/Agoric/documentation/pull/1163/commits/c190d502056c20833f2f57e0509874562e352240

**Pull request merged**